lib/web/fetch/body.js

```diff
@@ -318,7 +318,7 @@ function bodyMixinMethods (instance) {
       // given a byte sequence bytes: return a new ArrayBuffer
       // whose contents are bytes.
       return consumeBody(this, (bytes) => {
-        return new Uint8Array(bytes).buffer
+        return bytes.buffer
```

Comment:
You have to argue a little bit more. 
I guess consumeBody can only be done once, and so calling it multpile times is not possible, thus it is not necessary to wrap it in an Uin8Array to make the result immutable?

Reply:
Thanks, I was able to find that it wasn't cloned.  I have now fixed it.
This is not necessary since it has already been cloned here.
https://github.com/nodejs/undici/blob/54fa2b2138570a2db385fcc220f17fea818d6be0/lib/web/fetch/util.js#L1129-L1134